### PR TITLE
mysql: Add Spanish translation

### PR DESCRIPTION
**Pull request by ferrvic**
- [x] The page (if new), does not already exist in the repo.
- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#guidelines).
- [x] The page description includes a link to documentation or a homepage (if applicable).

### CODE REVIEW

File: pages.es/common/mysql.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
+# mysql
+
+> Herramienta en línea de comandos para gestionar bases de datos MySQL
```
Comment (navarroaxel) [inline pages.es/common/mysql.md:3]: Que yo recuerde, en español se usa `de`:
```suggestion
> Herramienta de línea de comandos para gestionar bases de datos MySQL
```

Comment (navarroaxel) [inline pages.es/common/mysql.md:3]: Que yo recuerde, en español se usa `de`:
```suggestion
> Herramienta de línea de comandos para gestionar bases de datos MySQL
```

File: pages.es/common/mysql.md [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+
+`mysql -h {{maquina_remota}} {{nombre_base_de_datos}}`
+
+- Conecta a una base de datos a través de un socket unix
+
+`mysql --socket {{ruta/al/socket.sock}}`
+
+- Ejecuta comandos SQL contenidos en un script:
+
+`mysql -e "source {{fichero.sql}}" {{nombre_base_de_datos}}`
```
Comment (navarroaxel) [inline pages.es/common/mysql.md:24]: Aunque en España se usa `fichero` o `archivo`. Mejor si usamos `archivo` que es lo más común en Latam:
```suggestion
`mysql -e "source {{archivo.sql}}" {{nombre_base_de_datos}}`
```

**Review by navarroaxel — CHANGES_REQUESTED** [review]
Hi @ferrvic, welcome and thanks for your submission. I left 1 comment.

**Comment by ferrvic** [comment]
I've applied the suggested changes. Thanks @navarroaxel!

**Review by navarroaxel — APPROVED** [review]

**Review by sbrl — APPROVED** [review]
Don't speak the language, but the syntax looks fine to me!

Thanks :slightly_smiling_face:

**Pull request merged**